File: sdk/java/examples/src/main/java/GetAccountBalanceExample.java
import com.hedera.hashgraph.sdk.AccountBalanceQuery;
import com.hedera.hashgraph.sdk.AccountId;
import com.hedera.hashgraph.sdk.Client;

class GetAccountBalanceExample {
    public static void main(String[] args) {
        var client = Client.forTestnet();

        var response = new AccountBalanceQuery()
            .setAccountId(AccountId.parse("0.0.1001"))
            .execute(client);

        System.out.printf("balance = %s\n", response.balance);
    }
}

File: sdk/kotlin/examples/src/main/kotlin/GetAccountBalanceExample.kt
@file:JvmName("GetAccountBalanceKtExample")

import com.hedera.hashgraph.sdk.AccountBalanceQuery
import com.hedera.hashgraph.sdk.AccountId
import com.hedera.hashgraph.sdk.Client

fun main() {
    val client = Client.forTestnet()

    val response = AccountBalanceQuery(
        accountId = AccountId.parse("0.0.1001")
    ).execute(client)

    println("balance = ${response.balance}")
}
